.github/workflows/linux_release.yml

```diff
@@ -75,6 +77,30 @@ jobs:
           path: ${{ runner.temp }}/shadow_build_dir/package/${{ env.ARTIFACT }}
           retention-days: 1
 
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-north-1
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+        if: startsWith(github.ref, 'refs/tags/')
+
+      - name: Build docker image
+        working-directory: ${{ runner.temp }}/shadow_build_dir
+        if: startsWith(github.ref, 'refs/tags/')
+        run:  |
+              docker build --build-arg APPIMAGE_PATH=${{ runner.temp }}/shadow_build_dir/package/${{ env.ARTIFACT }} ${SOURCE_DIR}/.github/workflows/Dockerfile -t ${ECR_REGISTRY}/${ECR_REPOSITORY}:latest ${ECR_REGISTRY/${ECR_REPOSITORY}:${{ github.ref }}
```

Comment:
I think "github.ref" translate to "refs/tags/v4.2.0-1.8.1" and similar, better to use "github.ref_name" whouch resolves to the tag name without the "refs/tags/" prefix

Reply:
Fixed.